play/manual/README.md

```diff
@@ -0,0 +1,32 @@
+# Manual Setup
+
+This directory contains a few scripts and corresponding config file templates to
+semi-manually setup and run a set of keypers locally. Basic instructions:
+
+0. Build `rolling-shutter` with `make -C ../../rolling-shutter build`.
+1. Run a postgres database, e.g. with `process-compose up` in `/play`
```

Comment:
Could you please also add the db directory initialization phase? I had to run `initdb -D work/db/` to successfully run `process-compose up`